superflore/docker.py

```diff
@@ -20,7 +22,7 @@
 class Docker(object):
     def __init__(self, dockerfile_directory, name):
         self.client = docker.from_env()
-        self.dockerfile_directory = dockerfile_directory
+        self.dockerfile_directory = os.path.dirname(dockerfile_directory)
```

Comment:
You're passing in the filename but calling it dockerfile_directory.  You should either change the variable name or pass in the dirname only and not run the dirname here.

Reply:
Good point. I think I'll change the parameter name to be `dockerfile`, and then open it's directory and set it to `self.dockerfile_directory`.